docs/IgorFromUA/games.html

```diff
@@ -0,0 +1,142 @@
+                            <option class="select__option" value="arpg">ARPG</option>
+                            <option class="select__option" value="fighting">Fighting</option>
+                            <option class="select__option" value="mmorpg">MMORPG</option>
+                            <option class="select__option" value="card">Card Game</option>
+                            <option class="select__option" value="action-rpg">Action RPG</option>
+                            <option class="select__option" value="open-world">Open-world</option>
+                            <option class="select__option" value="sports">sports</option>
+                            <option class="select__option" value="racing">racing</option>
+                            <option class="select__option" value="social">social</option>
+                            <option class="select__option" value="superhero">superhero</option>
+                            <option class="select__option" value="horror">horror</option>
+                            <option class="select__option" value="military">military</option>
+                            <option class="select__option" value="action">action</option>
+                            <option class="select__option" value="space">space</option>
+                            <option class="select__option" value="tank">tank</option>
+                            <option class="select__option" value="sailing">sailing</option>
+                            <option class="select__option" value="3d">3d</option>
+                            <option class="select__option" value="2d">2d</option>
+                        </select>
+                    </label>
+                    <label for="filter__platform">
+                        <select name="platform" id="filter__platform" data-filter-select="platform" class="filter__select">
+                            <option class="select__option" selected value="all">Platform</option>
+                            <option class="select__option" value="pc">PC</option>
+                            <option class="select__option" value="browser">Browser</option>
+                        </select>
+                    </label>
+                </div>
+                <div>
+                    <input type="radio" data-age-games="new" class="radio-input" name="age" id="age-1">
+                    <label for="age-1" class="radio-text    ">
+                        New first
+                    </label>
+                    <input type="radio" data-age-games="old" class="radio-input" name="age" id="age-2">
+                    <label for="age-2" class="radio-text">
+                        Old first
+                    </label>
+                </div>
+                <div class="filter-search-wrapper">
+                    <label for="filter__search">
+                        <input type="text" data-filter-search id="filter__search" class="filter__search"
```

Comment:
If a user hits `Enter` in this field, the whole page reloads, and all filters reset. So, proly, you need to handle `sumbit` event on form